src/client/Microsoft.Identity.Client/Internal/ClientCredentialWrapper.cs

```diff
@@ -96,18 +96,20 @@ public ClientCredentialWrapper(ApplicationConfiguration config)
 
             if (Certificate != null)
             {
-                Thumbprint = Base64UrlHelpers.Encode(Certificate.GetCertHash()); 
+                Thumbprint = Base64UrlHelpers.Encode(Certificate.GetCertHash());
             }
         }
 
         #region TestBuilders
         //The following builders methods are intended for testing
-        public static ClientCredentialWrapper CreateWithCertificate(X509Certificate2 certificate, IDictionary<string, string> claimsToSign = null)
+        public static ClientCredentialWrapper CreateWithCertificate(X509Certificate2 certificate, IDictionary<string, string> claimsToSign = null, bool withSendX5C=false)
         {
             ApplicationConfiguration applicationConfiguration = new ApplicationConfiguration();
+            
             applicationConfiguration.ClientCredentialCertificate = certificate;
             applicationConfiguration.ConfidentialClientCredentialCount = 1;
             applicationConfiguration.ClaimsToSign = claimsToSign;
+            applicationConfiguration.SendX5C = withSendX5C;
```

Comment:
I don't think this is used anywhere in the ClientCredentialWrapper?

Reply:
removed